package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.8.1",
+  "version": "2.9.1",
```

Comment:
If we're bumping the minor version we should reset the patch:

```suggestion
  "version": "2.9.0",
```

Reply:
Don't let me write code at 7am.

Reply:
Putting the parental controls on your work machine as we speak: "No VS Code before 7am"